### PR TITLE
Add nfc test index.html

### PR DESCRIPTION
**Pull request by kaixinjxq**

### CODE REVIEW

File: web-nfc/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<li><a href='./nfc/nfc_push_arraybuffer-manual.https.html'>nfc_push_arraybuffer-manual.https.html</a></li>
+<li><a href='./nfc/nfc_push_exceptions.https.html'>nfc_push_exceptions.https.html</a></li>
+<li><a href='./nfc/nfc_push_ignoreread_false-manual.https.html'>nfc_push_ignoreread_false-manual.https.html</a></li>
+<li><a href='./nfc/nfc_push_nfcmessage_json-manual.https.html'>nfc_push_nfcmessage_json-manual.https.html</a></li>
+<li><a href='./nfc/nfc_push_nfcmessage_opaque-manual.https.html'>nfc_push_nfcmessage_opaque-manual.https.html</a></li>
+<li><a href='./nfc/nfc_push_nfcmessage_text-manual.https.html'>nfc_push_nfcmessage_text-manual.https.html</a></li>
+<li><a href='./nfc/nfc_push_nfcmessage_url-manual.https.html'>nfc_push_nfcmessage_url-manual.https.html</a></li>
+<li><a href='./nfc/nfc_push_string-manual.https.html'>nfc_push_string-manual.https.html</a></li>
+<li><a href='./nfc/nfc_watch.https.html'>nfc_watch.https.html</a></li>
+<li><a href='./nfc/push-from-iframe.html'>push-from-iframe.html</a></li>
```
Comment (Honry) [inline web-nfc/index.html]: push-from-iframe.html is a resource rather then test case.

Comment (kaixinjxq) [inline web-nfc/index.html]: Need add resources folder really?

Comment (Honry) [inline web-nfc/index.html]: no, Just remove it from index.html

**Comment by Honry** [comment]
LGTM

**Pull request merged**